app/src/pages/inside/common/filterEntitiesGroups/launchLevelEntities.jsx

```diff
@@ -332,24 +333,29 @@ export class LaunchLevelEntities extends Component {
 
   getDynamicEntities = () => {
     const { defectTypes, intl, visibleFilters } = this.props;
-    return getGroupedDefectTypesOptions(defectTypes, intl.formatMessage).map((option) => {
-      return {
-        ...option,
-        id: option.value,
-        component: EntityInputConditional,
-        value: this.bindDefaultValue(option.value, {
-          condition: CONDITION_GREATER_EQ,
-        }),
-        validationFunc: commonValidators.launchNumericEntity,
-        title: option.label,
-        customProps: {
-          conditions: [CONDITION_GREATER_EQ, CONDITION_LESS_EQ, CONDITION_EQ],
-          placeholder: intl.formatMessage(messages.STATS_PLACEHOLDER),
-        },
-        removable: true,
-        active: visibleFilters.includes(option.value),
-      };
-    });
+    return getGroupedDefectTypesOptions(defectTypes, intl.formatMessage)
+      .filter(
+        (option) =>
+          option.groupId !== NO_DEFECT.toUpperCase() && option.groupRef !== NO_DEFECT.toUpperCase(),
```

Comment:
Why `&&`? It is an impossible case, because you'll get `undefined` for the first/second operand every time.
I guess it should be written via `||`.